src/CalcManager/UnitConverter.cpp

```diff
@@ -555,7 +562,7 @@ future<pair<bool, wstring>> UnitConverter::RefreshCurrencyRatios()
     return async([this, currencyDataLoader, sharedLoadResult]() {
         sharedLoadResult.wait();
         bool didLoad = sharedLoadResult.get();
-        wstring timestamp = L"";
+        wstring timestamp;
```

Comment:
No need to use the constructor taking a `const wchar_t*` when a default `wstring` is already an empty string.